### ISSUE TITLE
Should upgrade the operator_sdk.util version to "0.3.1" for the "k8s event" module

### ISSUE THREAD

**Issue by jmrodri** [operator-framework/operator-sdk#5461]
#### What did you do?

Generated an ansible-operator using operator-sdk: `operator-sdk init --plugins=ansible --domain=k8sevent.com`

#### What did you expect to see?
```
$ cat requirements.yml 
---
collections:
  - name: community.kubernetes
    version: "1.2.1"
  - name: operator_sdk.util
    version: "0.2.0"
  - name: kubernetes.core
    version: "2.2.0"
```
#### What did you see instead? Under which circumstances?

Expected to see a newer version of the operator-sdk.util

```
cat requirements.yml 
---
collections:
  - name: community.kubernetes
    version: "1.2.1"
  - name: operator_sdk.util
    version: "0.3.1"
  - name: kubernetes.core
    version: "2.2.0"
```

#### Environment

**Operator type:**

/language ansible

**Kubernetes cluster type:**

OpenShift and Kind

`$ operator-sdk version`

`operator-sdk version: "v1.15.0-8-g1d009d6a", commit: "1d009d6a36661f9beec45f3d9927da905428c800", kubernetes version: "v1.21", go version: "go1.16.8", GOOS: "linux", GOARCH: "amd64"`

`$ go version` (if language is Go)

`go version go1.16.8 linux/amd64`

`$ kubectl version`
```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:18:45Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.1+6859754", GitCommit:"2ade4d46986c94405aa0542f8d6ff7691bddb153", GitTreeState:"clean", BuildDate:"2021-12-03T16:26:49Z", GoVersion:"go1.17.2", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by asmacdo** [comment]
https://github.com/operator-framework/operator-sdk/pull/5462

**Comment by jmrodri** [comment]
Fixed by PR #5462
